src/client/components/ReservationForm.js

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from "react";
+
+function ReservationForm({ idmeals }) {
+  const [id, setId] = useState("");
+  const [createReservation, setCreateReservation] = useState({
+    number_of_guests: "",
+    meal_id: idmeals,
+    created_date: new Date().toISOString(),
+    contact_name: "",
+    contact_email: "",
+    phone_number: "",
+  });
+
+  const addReservation = async (createReservation) => {
```

Comment:
You don't need to pass in `createReservation` as this function already has access to that state.